deployment/hasura/migrations/AerieMerlin/2_drop_profile_topic_fks/down.sql

```diff
@@ -0,0 +1,75 @@
+-- profile_segment
+drop trigger if exists insert_update_profile_segment_trigger on profile_segment;
+drop function if exists profile_segment_integrity_function;
```

Comment:
I don't think we should have the "if exists" modifier. I believed we've discussed on a prior PR that if the database state is wrong when we're migrating, don't we want to be loud about it?

Reply:
I went through and dropped if exists if it exists

Reply:
How meta!